### PR TITLE
add_new_package: py-file-magic

### PR DESCRIPTION
**Pull request by Sinan81**
Note that this is a different package than `py-filemagic`

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-file-magic/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class PyFileMagic(PythonPackage):
+    """This library is a Python ctypes interface to libmagic"""
+
+    homepage = "https://pypi.org/project/file-magic/"
+    pypi     = "file-magic/file-magic-0.4.1.tar.gz"
+
+    version('0.4.1', sha256='a91d1483117f7ed48cd0238ad9be36b04824d57e9c38ea7523113989e81b9c53')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-file-magic/package.py]:
```suggestion
    depends_on('py-setuptools@61:', type='build')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by Sinan81** [comment]
@spackbot fix style

**Review by adamjstewart — APPROVED** [review]

**Comment by alalazo** [comment]
_One year and a half later_ Do you know where the sources are located for this package?

**Comment by alalazo** [comment]
Ok, these seems to come from https://github.com/file/file directly.

**Pull request merged**